pubsub-service-api.yml

```diff
@@ -152,26 +152,26 @@ paths:
       responses:
         "200":
           description: "Leave is effective"
-        '400':
-          $ref: '#/components/responses/BadRequest'
-        '401':
-          $ref: '#/components/responses/Unauthorized'
-        '404':
-          $ref: '#/components/responses/TopicNotFound'
-        '409':
-          $ref: '#/components/responses/InsufficientFunds'
-        '429':
-          $ref: '#/components/responses/TooManyRequests'
-        '4XX':
-          $ref: '#/components/responses/CustomServiceError'
-        '5XX':
-          $ref: '#/components/responses/InternalServerError'
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "404":
+          $ref: "#/components/responses/TopicNotFound"
+        "409":
+          $ref: "#/components/responses/InsufficientFunds"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+        default:
+          $ref: "#/components/responses/CustomServiceError"
```

Comment:
As discussed, I think we should remain with 4XX and 5XX to constraint where errors are located.